keras_hub/src/models/retinanet/retinanet_image_converter.py

```diff
@@ -10,25 +12,38 @@ class RetinaNetImageConverter(ImageConverter):
 
     def __init__(
         self,
-        image_size=None,
-        scale=None,
-        offset=None,
+        bounding_box_format,
+        pad_to_aspect_ratio=False,
```

Comment:
why do we need pad to aspect ratio here? and not for other models?

consider doing this on a separate PR, probably makes sense to do this on the base image converter class first if we want it

Reply:
Yup we need these because, crop to aspect ratio trims on other hand pad to aspect ratio preserves by adding black pixels.
Difference:

![image](https://github.com/user-attachments/assets/9a905a88-d591-449e-b49f-83b967933567)
![image](https://github.com/user-attachments/assets/58fe5aca-a1b0-4910-87a4-43881e0ad948)

Reply:
Sure make sense I will do that.